modules/installation-about-custom-aws-vpcs.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/
+
+[id="installation-about-custom-aws-vpcs_{context}"]
+= About using a custom VPC
+
+In {product-title} {product-version}, you can deploy a cluster into an existing VPC in Amazon Web Services (AWS). If you do, you must also use existing subnets within the VPC and routing rules.
```

Comment:
> If you do, you must also use existing subnets within the VPC and routing rules.

hmm, might need to be a little more focused :thinking:

Reply:
Do you have a specific suggestion to refocus this line?

Reply:
Instead of making this "you must also..." sounding like "this means you have to do more stuff" maybe we can make subnets part of the "you're here because you want to do this" portion with:

> In {product-title} {product-version}, you can deploy a cluster into existing subnets in an existing VPC in Amazon Web Services (AWS).  By deploying {product-title} into an existing AWS VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. This is a good option to use if you cannot obtain the infrastructure creation permissions that are required to create the VPC yourself.  Because the installer doesn't know what other components are also in your existing subnets, it cannot chose subnet CIDRs and so forth on your behalf, so you must configure networking for the provided subnets yourself.

The wording in my suggesting is still a bit glib, but maybe something in that order?  I'm also fine with the lead-in sentences as they stand in the PR now ;).